service/lib/dinstaller/dbus/software/proposal.rb

```diff
@@ -54,6 +55,7 @@ def initialize(logger)
           dbus_method :AddResolvables,
             "in Id:s, in Type:y, in Resolvables:as, in Optional:b" do |id, type, resolvables, opt|
             Yast::PackagesProposal.AddResolvables(id, TYPES[type], resolvables, optional: opt)
+            changed
```

Comment:
NP: This looks to me more like these bang methods. I mean, `change!` instead of `changed`.

Reply:
Or maybe a `call_on_change_callbacks`, which for sure is too long but more self-explanatory.

Reply:
well, I would definitively like `change!` as it emphasize to me that it change something which is not true. It is more about notification. the other one is a bit long for me. What about `notify_change` or with bang like `notify_change!`

Reply:
>I would definitively like `change!`

By the context, I guess you meant you don't like.

>  What about notify_change or with bang like notify_change!

Both looks better to me than `changed`. But maybe you can wait for another review to change (or not) it. More brains, more ideas :)